src/huggingface_hub/hf_api.py

```diff
@@ -4772,6 +4772,92 @@ def delete_file(
             parent_commit=parent_commit,
         )
 
+    @validate_hf_hub_args
+    def delete_files_r(
```

Comment:
```suggestion
    def delete_files(
```

I'd rename it `delete_files` instead (the `_r` is not very explicit)